### DISCUSSION TITLE
Unable to login to Facebook

### DISCUSSION THREAD

**Discussion by Sjsiisjsjskakkam** [thatskymod/Sky-CotL-Scripts#149] [Q&A]
I can't log in via Facebook and can't access my email. What should I do?

**Comment by thatskymod** [comment]
This issue duplicates #65, #144, #148, #149, and #157 and cannot be resolved without explicit action from the upstream developers. I understand the frustration caused by the Facebook login option appearing despite being dysfunctional. This is indeed a confusing design choice. However, kindly note that this repository is merely a collection of resources available to us, the end users, and we do not have the ability to address these issues ourselves. For bug reporting, it's best to contact the respective developers directly.

In the meantime, you might consider using the Nintendo login method as a workaround.
